zerver/lib/actions.py

```diff
@@ -4017,14 +4017,17 @@ def do_change_logo_source(
     send_event(realm, event, active_user_ids(realm.id))
 
 
-def do_change_plan_type(realm: Realm, plan_type: int) -> None:
+def do_change_plan_type(
+    realm: Realm, plan_type: int, acting_user: Optional[UserProfile] = None
+) -> None:
```

Comment:
Please use the style for this function detailed in https://github.com/zulip/zulip/pull/17025.